spec/bridgetown-inline-svg_spec.rb

```diff
@@ -79,7 +38,7 @@ def parse(f)
       end
 
       it "render site" do
-        expect(File.exist?(dest_dir("index.html"))).to be_truthy
+        expect(File.exist?(dest_dir("index.html"))).to eq(true)
```

Comment:
I didn't want to mess to much with the existing specs to I know they'd break as you'd expect.

The two failures right now are:

```
Failures:

  1) BridgetownInlineSvg Integration - without optimization - add a height if only width is given
     Failure/Error: expect(data[2].get_attribute("height")).to is_opt ? be_falsy : eql("")
     
       expected: ""
            got: "48"
     
       (compared using eql?)
     # ./spec/bridgetown-inline-svg_spec.rb:61:in `block (4 levels) in <top (required)>'

  2) BridgetownInlineSvg Integration - with optimization - add a height if only width is given
     Failure/Error: expect(data[2].get_attribute("height")).to is_opt ? be_falsy : eql("")
     
       expected: falsey value
            got: "48"
     # ./spec/bridgetown-inline-svg_spec.rb:61:in `block (4 levels) in <top (required)>'
``` 

This is because I made made it so height is always present with width is. Is this something optimised is intended for?

Reply:
I'm an idiot - I added a new SVG above the `data[2]` svg, throwing off the order. Adding the right order solved that.